File: python/kyu_7/disemvowel-trolls.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

# https://www.codewars.com/kata/52fba66badcd10859f00097e/train/python


def disemvowel(string_):
    vowels = ['a', 'e', 'i', 'o', 'u']
    for vowel in vowels + [v.upper() for v in vowels]:
        string_ = string_.replace(vowel, '')
    return string_

File: javascript/kyu_7/disemvowel-trolls.js
/*
 https://www.codewars.com/kata/52fba66badcd10859f00097e/train/javascript
*/

const disemvowel = (str) => {
    const vowels = new Set(['a', 'e', 'i', 'o', 'u'])
    return str.split('').filter((c) => !vowels.has(c.toLowerCase())).join('')
}
